src/styles/blocks/head-block/nav.scss

```diff
@@ -0,0 +1,50 @@
+.nav {
+  &__list {
+    display: flex;
+    padding: 0;
+  }
+
+  &__item {
+    list-style: none;
+    display: flex;
+  }
+
+  &__item:not(:last-child) {
+    margin-right: 20px;
+  }
+
+  &__item a {
+    display: inline-block;
+    text-decoration: none;
+    font-size: 12px;
+    font-weight: 500;
+    height: 60px;
+    line-height: 60px;
+    color: $nav_text-color;
+    cursor: pointer;
+    text-transform: uppercase;
+
+    &.is-active {
+      color: $main-blue-color;
+    }
+
+    &.is-active::after {
+      content: "";
+      position: absolute;
+      bottom: 0;
+      display: block;
+      height: 4px;
+      width: 37px;
+      border-radius: 5px;
+      background-color: $main-blue-color;
+    }
+  }
+
+  &--is-active a:visited {
+    color: $main-blue-color;
+  }
+
+  &__item a:hover {
```

Comment:
Smooth hover effect is missing. You can achieve this by using the transition property. For example: 'transition: color 0.3s ease;'.